dojson/overdo.py

```diff
@@ -55,7 +55,9 @@ def do(self, blob):
 
         for key, value in iteritems(blob):
             for name, creator, field in self._query(key):
-                output[name] = creator(output, key, value)
+                val = creator(output, key, value)
```

Comment:
That sounds cool. I will try this approach. Do you know how to run a simple test:

    py.test tests

does not work.